domains/transfer-agreements/TransferAgreements.API/API/Options/OtlpOptions.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Options;
+
+public class OtlpOptions
+{
+    public const string Prefix = "Otlp";
+
+    [Required]
+    public Uri ReceiverEndpoint { get; init; } = null!;
```

Comment:
If I set the others to init tests brake, since they are set in test constructors. If that is needed for this some day, this can be set to 'set' as well. Or do you want me to set it to 'set' now?

Reply:
I think that the Options-classes should be consistent. It is up to you if it is get/init or get/set.